packages/f-checkout/src/components/tests/Selector.test.js

```diff
@@ -10,6 +10,17 @@ const localVue = createLocalVue();
 localVue.use(VueI18n);
 localVue.use(Vuex);
 
+const fulfillmentTimes = [
```

Comment:
If you're setting this multiple times in different files (as the same thing), you could put it into the constants file.